FLEDGE_browser_bidding_and_auction_API.md

```diff
@@ -466,7 +476,11 @@ Prior to compression and encryption, the AuctionResult is encoded as CBOR with t
       "description": "Boolean to indicate that there is no remarketing winner from the auction. AuctionResult may be ignored by the client (after decryption) if this is set to true."
     },
     "winReportingUrls": { "$ref": "#/$defs/winReportingUrlsDef" },
-    "error": { "$ref": "#/$defs/errorDef" }
+    "error": { "$ref": "#/$defs/errorDef" },
+    "nonce": {
+      "type": "byte string",
```

Comment:
The nonce probably could be a text string, to make it easier to match against the header.

Reply:
SGTM.  I was gonna say that it's easy to turn the binary into text, but perhaps executing [the code to turn the binary into the internal type](https://source.chromium.org/chromium/chromium/src/+/main:base/uuid.cc;l=89;drc=24386822ca7fb24c3785efb4d709f137c1b8f786) could be considered [unsafe parsing](https://chromium.googlesource.com/chromium/src/+/master/docs/security/rule-of-2.md), so let's just keep it all text.  It'll make the B&A response size 36 bytes bigger but we could always improve that later and response size is much less critical than request size.  cc @qingxinwu

Reply:
Resolving as I updated the PR.